### PR TITLE
Improve Maven structure of whole project

### PR DESCRIPTION
**Pull request by jkosternl**
* Introduce dependencyManagement section
* Define submodule versions only in the parent pom
* Enhance parent pom with project information

### CODE REVIEW

File: sandbox/pom.xml
Comment (preetamnpr) [inline sandbox/pom.xml:26]: Could you adjust the indentation ? as the above pom files are indented properly, but rest of the below files are not.

**Review by preetamnpr — CHANGES_REQUESTED** [review]

File: sandbox/pom.xml
Comment (jkosternl) [reply sandbox/pom.xml:26]: Yeah... I was in doubt to fix all those things as well. Shall I reformat all pom.xml files, after I got approvals (not messing up the diff, while reviewing)

Comment (preetamnpr) [reply sandbox/pom.xml:26]: I approve it, please reformat the poms.

**Review by nt-gt — APPROVED** [review]

**Review by palatsangeetha — APPROVED** [review]

**Pull request merged**
